airbyte-integrations/connectors/source-stripe/acceptance-test-config.yml

```diff
@@ -1,50 +1,67 @@
 connector_image: airbyte/source-stripe:dev
-tests:
+test_strictness_level: high
+acceptance_tests:
   spec:
+    tests:
     - spec_path: "source_stripe/spec.yaml"
   connection:
-    - config_path: "secrets/config.json"
-      status: "succeed"
-    - config_path: "secrets/connected_account_config.json"
-      status: "succeed"
-    - config_path: "integration_tests/invalid_config.json"
-      status: "failed"
+    tests:
+      - config_path: "secrets/config.json"
+        status: "succeed"
+      - config_path: "integration_tests/invalid_config.json"
+        status: "failed"
   discovery:
-    - config_path: "secrets/config.json"
-      backward_compatibility_tests_config:
-        disable_for_version: "2.0.0"
-    - config_path: "secrets/connected_account_config.json"
-      backward_compatibility_tests_config:
-        disable_for_version: "2.0.0"
+    tests:
+      - config_path: "secrets/config.json"
+        backward_compatibility_tests_config:
+          disable_for_version: "2.0.0"
```

Comment:
Why is this version incompatible, but not a breaking (major semver) change?

Reply:
As I see from previous commits there was an invoices schema fix. Some field types were changed: `number` -> `integer`, `integer` -> `string`. That is why we disabling this compatibility test

Reply:
I guess those changes are scope-increasing, so that should be OK